internal/compiler/parser/expressions.rs

```diff
@@ -417,3 +412,80 @@ fn parse_tr(p: &mut impl Parser) {
     }
     p.expect(SyntaxKind::RParent);
 }
+
+/// ```test,AtTr
```

Comment:
```suggestion
/// ```test,AtImageUrl
```

Reply:
I also forgot the `#[cfg_attr(test, parser_test)]` which is why it did not fail

```suggestion
#[cfg_attr(test, parser_test)]
/// ```test,AtImageUrl
```